File: src/main/java/com/back/domain/member/review/entity/Review.java
package com.back.domain.member.review.entity;

import com.back.domain.member.member.entity.Member;
import com.back.global.jpa.entity.BaseEntity;
import jakarta.persistence.Entity;
import jakarta.persistence.ManyToOne;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Entity
@Getter
@NoArgsConstructor
public class Review extends BaseEntity {
    @ManyToOne
    private Member member;
    private String comment;
    private int star;
    private int reviewerId;

    public Review(Member member, int star, String comment, int reviewerId) {
        this.member = member;
        this.star = star;
        this.comment = comment;
        this.reviewerId = reviewerId;
    }
}

File: src/main/java/com/back/domain/member/review/entity/Review.kt
package com.back.domain.member.review.entity

import com.back.domain.member.member.entity.Member
import com.back.global.jpa.entity.BaseEntity
import jakarta.persistence.Entity
import jakarta.persistence.ManyToOne
import lombok.Getter
import lombok.NoArgsConstructor

@Entity
@Getter
@NoArgsConstructor
class Review(
    @field:ManyToOne var member: Member,
    var star: Int,
    var comment: String?,
    var reviewerId: Int
) : BaseEntity()
